src/gt4py/frontend/gtscript_frontend.py

```diff
@@ -1230,7 +1230,7 @@ def collect_external_symbols(definition):
         imported_symbols = {name: {} for name in imported_names}
 
         context, unbound = gt_meta.get_closure(
-            definition, included_nonlocals=False, include_builtins=False
+            definition, included_nonlocals=True, include_builtins=False
```

Comment:
`True` is needed here to be able to include the value of `A` in `inspect.closurevars` when importing the function (which is using an external `A`) as an external.

Reply:
cc: @tehrengruber

Reply:
I can't tell anymore but it feels to me that this change could affect the current behavior of the frontend in other places. Have you considered and tested all the implications? @tehrengruber do you also agree this is a safe change?